.github/labeler.yml

```diff
@@ -0,0 +1,22 @@
+#Add "2023" to any change to 2023 project files
+
+2023:
```

Comment:
Does 2023 need to be wrapped in quotes?

Reply:
If the label identifier don't contains space or any special character, quotes are optional.
[common example link](https://github.com/actions/labeler#common-examples)

Reply:
It is only a nit but I'd say change the 

`#Add "2023" to any change to 2023 project files`
to
`# Add '2023' to any change to 2023 project files`

so that it is consistent with the  rest of the comments in the file. 

The difference is a distraction and as the other comments use single quotes then the double quoted 2023 reads as, _so-called 2023_